crt_portal/static/sass/custom/modal.scss

```
@@ -85,3 +103,26 @@ body.is-modal {
     white-space: pre-wrap;
   }
 }
+
+#external-link--modal {
+  h1 {
+    @include text--heading__medium();
+    font-weight: $theme-font-weight-bold;
+    color: color($theme-color-primary-darker);
+    margin: 0;
+  }
+
+  .external-link--disclaimer {
+    font-style: italic;
+    font-size: 14px;
```

Could we use the variable instead, pretty please?
```suggestion
    font-size: $theme-text-font-size-sm;
```

---

Right you are :D